app/resources/v1/annotation_resource.rb

```diff
@@ -13,10 +13,10 @@ module V1
   #
   # @example
   #
-  #   curl -X GET http://localhost:3000/v1/annotations/1
-  #   curl -X GET http://localhost:3000/v1/annotations/
+  #   curl -X GET http://localhost:3100/v1/annotations/1
+  #   curl -X GET http://localhost:3100/v1/annotations/
   #
-  #   curl -X POST "http://yourdomain.com/v1/annotations" \
+  #   curl -X POST "http://localhost:3100/v1/annotations" \
   #     -H "accept: application/vnd.api+json" \
```

Comment:
There is currently no route for top level annotations so I think we can modify accordingly.